app/connectors/CustomsFinancialsApiConnector.scala

```diff
@@ -91,6 +92,18 @@ class CustomsFinancialsApiConnector @Inject()(httpClient: HttpClientV2,
                               (implicit hc: HeaderCarrier): Future[Either[ErrorResponse, CashTransactions]] = {
     val cashDailyStatementRequest = CashDailyStatementRequest(can, from, to)
 
+    def addUUIDToCashTransaction(response: CashTransactions): CashTransactions = {
```

Comment:
Can this be made private?

Reply:
**addUUIDToCashTransaction** is already in a local scope so it can't be made a private method